src/pages/home/report/ReportActionItem.js

```diff
@@ -325,6 +329,8 @@ class ReportActionItem extends Component {
                                                 action={this.props.action}
                                                 showHeader={!this.props.draftMessage}
                                                 wrapperStyles={[styles.chatItem, isWhisper ? styles.pt1 : {}]}
+                                                shouldShowSubscriptAvatar={this.props.shouldShowSubscriptAvatar}
+                                                report={this.props.report}
```

Comment:
@aimane-chnaif can you submit a screenshot of where you think the SubscriptAvatar should be displayed as well? Just trying to keep things consistent since a lot of these screens will be updated/removed.